package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@fishtank/type",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Do not manually update the version - the version will be bumped during the deploy from master.